ow_plexil/src/plans/HealthMonitor.plp

```diff
@@ -0,0 +1,98 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// This plan monitors the faultDependencies of each subsystem. When a 
+// subsystem becomes inoperable, it prints out a message with the status and cause.
+// Other plans are intended to use the HealthMonitor with the InOut Boolean variables
+// that are updated every second. This is intended to make monitoring faults easiera and
+// less verbose in new plans. 
+
+#include "plan-interface.h"
+
+HealthMonitor:
+{
+  InOut Boolean AllOperable = true;
+  InOut Boolean ArmOperable = true;
+  InOut Boolean AntennaOperable = true;
+  InOut Boolean CameraOperable = true;
+  InOut Boolean PowerOperable = true;
+
+  Boolean PrevArm = true;
+  Boolean PrevAntenna = true;
+  Boolean PrevCamera = true;
+  Boolean PrevPower = true;
+  
+  LookupLoop:
+  {
+    Repeat true;
+    Wait 1;
+
+    ArmOperable = Lookup(IsOperable("Arm"));
+    AntennaOperable = Lookup(IsOperable("Antenna"));
+    CameraOperable = Lookup(IsOperable("Camera"));
+    PowerOperable = Lookup(IsOperable("Power"));
+
+    AllOperable = ArmOperable && 
+                  AntennaOperable &&
+                  CameraOperable &&
+                  PowerOperable;
+
+    if !ArmOperable && PrevArm{
+      if Lookup(IsFaulty("Arm")){
+        log_info("Arm is Inoperable due to the local fault(s): ", Lookup(ActiveFaults("Arm")));
+      }
+      else{
+        log_info("Arm is Inoperable due to non-local fault(s).");
+      }
+      PrevArm = false;
+    }
+    if ArmOperable && !PrevArm{
+        log_info("Arm is now operable.");
+        PrevArm = true;
+    }
```

Comment:
I think the four similar instances of this code block could be replaced with a Library node of 3 parameters.  Optional since you are pressed for time!